lib/ruby_lsp/ruby_lsp_rails/addon.rb

```diff
@@ -32,6 +32,12 @@ def initialize
         @rails_runner_client = T.let(NullClient.new, RunnerClient)
         @global_state = T.let(nil, T.nilable(GlobalState))
         @outgoing_queue = T.let(nil, T.nilable(Thread::Queue))
+        @settings = T.let(
+          {
+            enable_pending_migration_prompt: true,
```

Comment:
Should it be `enablePendingMigrationPrompt` instead? Right now config options like `rubyLsp.indexing`'s keys are all camelcases and I think that'd match most VS Code setting in general too.

Reply:
Also, I think it should be pluralized, to match the button shown in the browser.